src/FSharp.Control.AsyncSeq/AsyncSeq.fs

```diff
@@ -1344,6 +1344,13 @@ module AsyncSeq =
           let! moven = ie.MoveNext()
           b := moven }
 
+  let head (source : AsyncSeq<'T> = async {
```

Comment:
These also need to be added to the signature file please

Reply:
@dsyme Fixed.